lib/fathead/perl_doc/Parsers/parse.pl

```diff
@@ -739,6 +747,13 @@ sub parse_page {
     }
 }
 
+sub add_related {
+    my ( $self, $article, @related ) = @_;
+    my $sql = 'UPDATE output.txt SET related = ? WHERE title = ?';
+    my $related = join ",", map { "[[$_]]" } @related;
```

Comment:
@jdorweiler Hmmm, are related articles comma separated or newline?

Doesn't appear to be in the docs. Thanks!